requirements.txt

```diff
@@ -1,4 +1,3 @@
-Flask>=1.0.0
-flask-cors>=3.0.8
```

Comment:
We still need flask-cors to allow CORS because front-end development and production env will be from different origin.